ulimit.go

```diff
@@ -4,6 +4,7 @@ import (
 	"fmt"
 	"strconv"
 	"strings"
+	"syscall"
```

Comment:
Should probably be `golang.org/x/sys`, given that "syscall" is frozen/deprecated in favour of that package